crates/bevy_plugin/src/dialogue_runner.rs

```diff
@@ -312,13 +312,13 @@ impl DialogueRunner {
     /// Returns a struct that can be used to access a portion of the underlying [`Dialogue`]. This is advanced functionality.
     #[must_use]
     pub fn inner(&self) -> InnerDialogue {
-        InnerDialogue(&self.dialogue)
+        InnerDialogue(self.dialogue.as_ref().unwrap())
```

Comment:
Could you add the standard "Some weird error occured, this is a bug, plz report it" text that is on `expect`s throughout the code here as well? This comment goes for all `.unwrap()`s

Reply:
Plz use the exact same string we already have in place

Reply:
Ah, sorry, should have checked the other messages. I think it's consistent now. I used a custom intro with the standard second half like other code seems to.